From 12895730675255e8e9b1cdca761cd21dcb771c12 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Thu, 31 Mar 2016 09:27:40 +0800
Subject: [PATCH] ARM64: rockchip_cros_defconfig: add the gpio regulator

The gru/kevin board works the iodomain with gpio regulator.
That's useful for rk3399 kevin/gru board.

Change-Id: I70b6185e3a21a038347b9f8ccd679908817184dd
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 8d100d4e06f1..42c66a061054 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -329,6 +329,7 @@ CONFIG_REGULATOR_DEBUG=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_ACT8865=y
 CONFIG_REGULATOR_FAN53555=y
+CONFIG_REGULATOR_GPIO=y
 CONFIG_REGULATOR_MP8865=y
 CONFIG_REGULATOR_RK808=y
 CONFIG_REGULATOR_RK818=y
-- 
2.35.3

